docs/tactical/STORE.md

```diff
@@ -0,0 +1,50 @@
+## Tactical Store API
```

Comment:
I'm wondering if we should avoid doing this until we can auto-generate documentation - I think it will just fall out of date otherwise.